Rakefile

```diff
@@ -4,3 +4,6 @@
 require_relative "config/application"
 
 Rails.application.load_tasks
+
+Rake::Task["assets:precompile"].clear
```

Comment:
NICE!

Reply:
Let's chat about this one next time we meet!